metadata-service/factories/src/main/resources/application.yml

```diff
@@ -34,14 +34,6 @@ authorization:
     enabled: ${AUTH_POLICIES_ENABLED:true}
     cacheRefreshIntervalSecs: ${POLICY_CACHE_REFRESH_INTERVAL_SECONDS:120}
 
-  # Optional: A set of custom authorizers, serving in addition to the default DataHub policies-based authorizer.
```

Comment:
What if someone is using this? I guess the assumption is that we are safe to remove this old code path? 

Is there a way to keep _both_ ways to register an authorizer? 

Only worried if people have already created their own. At minimum, we need a doc and a clear announcement that this will change